README.zh-CN.md

```diff
@@ -61,23 +61,198 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): 使用 AsyncAPI 进行事件模式管理的目录服务。
 - [EventMesh-go](https://github.com/apache/eventmesh-go): EventMesh 运行时的 Go 语言实现。
 
-## 快速开始
+## 快速入门   
 
-指南:
+本节指南将指导您分别从[本地](#在本地运行-eventmesh-runtime)、[docker](#在-docker-中运行-eventmesh-runtime-)、[k8s](#在-kubernetes-中运行-eventmesh-runtime-)部署EventMesh的步骤:
+
+本节指南只是帮助您快速入门EventMesh部署，按照默认配置启动EventMesh，如果您需要更加详细的EventMesh部署步骤，请访问[EventMesh官方文档](https://eventmesh.apache.org/docs/next/introduction)。
+
+### 部署 EventMesh Store  
+
+> EventMesh现在支持`standalone`、`RocketMQ`、`Kafka`等中间件作为存储   
+> 如果是在非`standalone`模式下，需要先部署所需的`store`，以`rocketmq`模式为例: 部署[RocketMQ](https://rocketmq.apache.org/docs/quickStart/01quickstart/)
 
-[第1步: 部署EventMesh Store](https://eventmesh.apache.org/docs/instruction/store)
+### 在本地运行 EventMesh Runtime
 
-[第2步: 启动EventMesh Runtime](https://eventmesh.apache.org/docs/instruction/runtime)
+请在开始之前检查JKD版本，需要下载Java 8.
+```
+$ java -version
+java version "1.8.0_311"
+```
 
-[第3步: 运行示例](https://eventmesh.apache.org/docs/instruction/demo)
+#### 1.下载
 
-另外，如果您更喜欢使用Docker，则我们还为您提供了Docker版本的指南：
+在[EventMesh download](https://eventmesh.apache.org/download/)页面选择所需要版本的Binary Distribution进行下载,您将获得`apache-eventmesh-1.10.0-bin.tar.gz`。
```

Comment:
Please use Chinese punctuations in CN doc and and whitespaces after EN punctuations.

I think writting like this will be more concise:

![image](https://github.com/apache/eventmesh/assets/41445332/305ce345-d0ea-4d8f-9b3f-6d36f30ae069)